joomla/cassiopeia
# [Testing] This is a fake PR for testing different work pieces together

Pull Request for Issue # .

### Summary of Changes

This PR is just a fake PR for testing, i.e. it will not be merged.

It brings together the changes from following PRs and their branches:
- PR #18 "Use local Google Fonts"- branch [4.0-dev-google-fonts-enhancements](https://github.com/joomla/cassiopeia/tree/4.0-dev-google-fonts-enhancements)
- PR #19 "Sample data for Cassiopeia header" - branch [4.0-dev-sample-data-enhancement](https://github.com/joomla/cassiopeia/tree/4.0-dev-sample-data-enhancement)
- PR #46 "Add Metismenu to Cassiopeia, rework Cassiopeia's HTML structure, use relative CSS sizes" - branch [4-0-dev-frontend-metismenu-and-html-structure](https://github.com/joomla/cassiopeia/tree/cassiopeia/4-0-dev-frontend-metismenu-and-html-structure)

Developers of these features, please continue to develop on the branch(es) mentioned above for your work piece(s).

For updating this branch here with your changes for testing, merge then the particular branch of your workpiece into the branch [4.0-dev-testing-metismenu-sampledata-htmlstructure](https://github.com/joomla/cassiopeia/tree/4.0-dev-testing-metismenu-sampledata-htmlstructure) of this fake PR.

How this is to be done depends on your Git client, but normally (e.g. in GitHub Desktop) you check out the branch of this fake PR and then select "Merge into current branch ..." in the menu, and then chose the branch of your workpiece.

### Testing Instructions

Checkout the branch, pull the latest changes and run "npm ci".

Or use the following package to make a new installation:

[https://test5.richard-fath.de/Joomla_4.0.0-beta4-dev-Cassiopeia-Test-Full_Package_2020-08-29_v1.zip](https://test5.richard-fath.de/Joomla_4.0.0-beta4-dev-Cassiopeia-Test-Full_Package_2020-08-29_v1.zip)

Or use the following package to update a current 4.0-dev or latest 4.0 nightly build from 2020-08-29 or a bit earlier, but not too old:

[https://test5.richard-fath.de/Joomla_4.0.0-beta4-dev-Cassiopeia-Test-Update_Package_2020-08-29_v1.zip](https://test5.richard-fath.de/Joomla_4.0.0-beta4-dev-Cassiopeia-Test-Update_Package_2020-08-29_v1.zip)

After that, login to backend and install "Blog Sample Data" to see the metismenu being used in frontend.
